### PR TITLE
Merge latest changes for W-like unblinding of helicity cross section fit

### PR DESCRIPTION
**Pull request by emanca**

### CODE REVIEW

File: wremnants/combine_theory_helper.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def add_resum_unc(self, scale=1):
         if self.minnlo_unc and self.minnlo_unc not in ["none", None]:
             # sigma_-1 uncertainty is covered by scetlib-dyturbo uncertainties if they are used
             helicities_to_exclude = None if self.resumUnc == "minnlo" else [-1]
-            for sample_group in self.samples:
+            for sample_group in ["signal_samples_inctau","single_v_nonsig_samples"]:
```
Comment (davidwalter2) [inline wremnants/combine_theory_helper.py:148]: What is the reason for this change?

File: wremnants/combine_theory_helper.py [hunk 2]
```diff
@@ -564,9 +564,7 @@ def add_pdf_uncertainty(self, operation=None, scale=-1.):
         pdf_ax = self.syst_ax if self.pdf_from_corr else "pdfVar"
         symHessian = pdfInfo["combine"] == "symHessian"
 
-        processesZ = [] if self.skipFromSignal else ['single_v_samples']
-        processesW = ['wtau_samples', 'single_v_nonsig_samples'] if self.skipFromSignal else ['single_v_samples']
-        processes = processesW if self.label=="W" else processesZ
+        processes = ['single_v_samples']
```
Comment (davidwalter2) [inline wremnants/combine_theory_helper.py:567]: Also here, this seems to remove some functionality. Is this change needed?

File: wremnants/combine_theory_helper.py (hunk 1 above)
Comment (emanca) [reply wremnants/combine_theory_helper.py:148]: This is needed to apply the QCD scale variations to signal in the theory agnostic version of the fit

File: wremnants/combine_theory_helper.py (hunk 2 above)
Comment (emanca) [reply wremnants/combine_theory_helper.py:567]: Same as up. But for PDFs

File: wremnants/combine_theory_helper.py (hunk 1 above)
Comment (emanca) [reply wremnants/combine_theory_helper.py:148]: For the standard analysis nothing changes in principle, can check with the CI

File: wremnants/combine_theory_helper.py (hunk 2 above)
Comment (davidwalter2) [reply wremnants/combine_theory_helper.py:567]: But if skipFromSignal is false you should get the same as before. It's just that the new implementation removes the skipFromSignal functionality, maybe this was needed for some tests or so. Maybe @kdlong know?

File: wremnants/combine_theory_helper.py (hunk 1 above)
Comment (davidwalter2) [reply wremnants/combine_theory_helper.py:148]: For the standard analysis yes, but we also want to keep running alternative configurations. The change seems to me to make things less flexible.

Comment (bendavid) [reply wremnants/combine_theory_helper.py:148]: Wouldn't this add QCD scale variations to other samples aside from the W and Z? (and also change the splitting for the W and Z samples?)

Comment (bendavid) [reply wremnants/combine_theory_helper.py:148]: (I would suggest looking carefully to make sure the expected number/splitting of nuisances are there)

**Comment by bendavid** [comment]
From the CI it looks like there are some changes to the nuisance groups also for the standard analysis.  Is it expected? (and can you comment what's changed here?)

File: wremnants/combine_theory_helper.py (hunk 1 above)
Comment (bendavid) [reply wremnants/combine_theory_helper.py:148]: So actually I checked this explicitly for the standard analysis, and this change keeps the behaviour unchanged, but is actually more transparent about what is done and less error-prone, so I would actually keep it like this.

File: wremnants/combine_theory_helper.py (hunk 2 above)
Comment (bendavid) [reply wremnants/combine_theory_helper.py:567]: Yes I agree that this change can simply be reverted and the helicity fit should run fine, since as David said unless you use skipFromSignal for some reason there is no actual change here.

Comment (bendavid) [reply wremnants/combine_theory_helper.py:567]: Ok so after discussing with @emanca I understand better that the skipFromSignal option exists precisely for the helicity fit, and the change in behaviour here is intentional (to add back these uncertainties even when the option is enabled)

So this can also stay as it is.

**Comment by bendavid** [comment]
Ok now the CI only changes for the helicity xsec fit as expected.
